### PR TITLE
feat: install script

### PR DESCRIPTION
**Pull request by technicallyty**
- adds an install script
- updates docs with curl command to execute remote script
- updates docs for the `From Source` option to checkout latest tag instead of using main

### CODE REVIEW

**Comment by technicallyty** [comment]
Please help me test this!!!

run this:
```
curl -sSL https://raw.githubusercontent.com/skip-mev/slinky/tyler/blo-1525-curl-command-for-installing-connect/scripts/install.sh | sudo bash
```

**Comment by aljo242** [comment]
works for me! 

do we want to call it `slinky` or `slinkyd`?

**Review by aljo242 — APPROVED** [review]

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+sudo mv "${SLINKY_BIN}" /usr/local/bin/
+
+# Make it executable
+sudo chmod +x /usr/local/bin/slinky
+
+# Clean up
+rm -rf "${TEMP_DIR}"
+rm "${FILE_NAME}"
+
+echo "Slinky ${VERSION} has been installed successfully!"
```
Comment (aljo242) [inline scripts/install.sh:80]: could have another output that is just like `run slinky --help for more info` 

but not big deal really

File: codecov.yml [hunk 2]
```diff
@@ -1,2 +1,3 @@
 ignore:
-  - ".github/**/*"
+  - "**/*.*"
+  - "!**/*.go"
```
Comment (Eric-Warehime) [inline codecov.yml]: We should probably add some additional exclusions like `pulsar`, `pb`, etc.

**Pull request merged**